### DISCUSSION TITLE
[NOVA TRILHA] CSS3

### DISCUSSION THREAD

**Discussion by LuizFernando-TC** [codaqui/institucional#71] [Trilhas de Ensino]
Issue: https://github.com/codaqui/institucional/issues/24
Vamos adicionar uma nova trilha na Codaqui?
Sugiro criamos uma trilha de CSS3, uma linguagem de marcação importante no ambiente web para criar estilos.

**Comment by LuizFernando-TC** [comment]
Issue: https://github.com/codaqui/institucional/issues/24

**Comment by endersonmenezes** [comment]
Trilha aprovada, vamos discutir tecnicamente ela com colaboração de pessoas na Issue mesmo o/
